packages/edit-site/src/components/template-details/index.js

```diff
@@ -101,23 +90,11 @@ export default function TemplateDetails( { template, onClose } ) {
 
 			<Button
 				className="edit-site-template-details__show-all-button"
-				{ ...( newMenuSidebar
-					? {
-							href: addQueryArgs( 'edit.php', {
-								// TODO: We should update this to filter by template part's areas as well.
-								post_type: template.type,
-							} ),
-					  }
-					: {
-							onClick: showTemplateInSidebar,
-							'aria-label': sprintf(
-								/* translators: %1$s: the template part's area name ("Headers", "Sidebars") or "templates". */
-								__(
-									'Browse all %1$s. This will open the %1$s menu in the navigation side panel.'
-								),
-								templateSubMenu.title
-							),
-					  } ) }
+				href={ addQueryArgs( '', {
+					page: 'gutenberg-edit-site',
+					// TODO: We should update this to filter by template part's areas as well.
+					postType: template.type,
```

Comment:
It's typical in WordPress that URL query params use `snake_case`.

Reply:
It has already been using camelCase before even having the focus mode, I'm not sure why it's the case though.

Reply:
Oh I see. Let's leave it then.